### ISSUE TITLE
No data shown on an entry, neither in listview or normal

### ISSUE THREAD

**Issue by rasmusfjord** [TimGeyssens/UIOMatic#111]
So i have a model with UI O Matic.

But when it its shown the data from the field is not shown in the field, the field is just empty. If i fill it out, save it, it will actually get saved to the DB, when i reload the browser the fields are empty again.

```
    [TableName(TableName)]
    [PrimaryKey(PrimaryKey, autoIncrement = true)]
    [ExplicitColumns]
    [UIOMatic("teamMembers", "TeamMembers", "TeamMember", FolderIcon = "icon-users", ItemIcon = "icon-user")]
    public class TeamMember
    {
        #region Constants
        public const string TableName = "TeamMembers";
        public const string PrimaryKey = "Id";
        #endregion

        #region Properties

        [Column(PrimaryKey)]
        [PrimaryKeyColumn(AutoIncrement = true)]
        [JsonProperty("id")]
        public int Id { get; set; }

        [Column("TeamId")]
        [JsonProperty("teamId")]
        public int TeamId { get; set; }

        [Column("OrderId")]
        [JsonProperty("orderId")]
        public string OrderId { get; set; }

     
        [Column("Name")]
        [JsonProperty("name")]
        [UIOMaticField(Name = "Navn", View = UIOMatic.Constants.FieldEditors.Textfield)]
        public string Name { get; set; }

       
        [Column("Address")]
        [JsonProperty("address")]
        [UIOMaticField(Name = "Adresse", View = UIOMatic.Constants.FieldEditors.Textfield)]
        public string Address { get; set; }
}
```¨

Any idea why?

**Comment by TimGeyssens** [comment]
Hmmm need to check might be due to the JsonProperty attributes, that's the only thing I can see that is additional to our docs

**Comment by rasmusfjord** [comment]
It was because of the json declaration.

Any think you think we can do so they can coexist :)

**Comment by TimGeyssens** [comment]
Yeah will need to be updated in our code :) will try to take a look soon

**Comment by rasmusfjord** [comment]
Roger ;) 

Already on listview type.

Med venlig hilsen
----
Rasmus Fjord

42407620

On 24-03-2017 13:55:27, Tim Geyssens <notifications@github.com> wrote:
Yeah will need to be updated in our code :) will try to take a look soon

**Comment by TimGeyssens** [comment]
Fixed in upcoming v3.10.0
